src/main/java/org/reaktivity/nukleus/http2/internal/Http2Connection.java

```diff
@@ -144,7 +147,9 @@
         decodeContext = new HpackContext(localSettings.headerTableSize, false);
         encodeContext = new HpackContext(remoteSettings.headerTableSize, true);
         http2Writer = factory.http2Writer;
-        writeScheduler = new Http2WriteScheduler(this, sourceOutputEstId, factory.frameSlab, networkConsumer,
+        frameSlab = factory.bufferPool.duplicate();
+        headersSlab = factory.bufferPool.duplicate();
```

Comment:
These calls to `duplicate` appear to be per-connection, but are only needed once in the factory and can then be shared across all connections, right?